modules/oc-mirror-support.adoc

```diff
@@ -10,3 +10,9 @@
 The oc-mirror plugin supports mirroring {product-title} payload images and Operator catalogs for {product-title} versions 4.9 and later.
 
 Use the latest available version of the oc-mirror plugin regardless of which versions of {product-title} you need to mirror.
+
+// TODO: Remove this in 4.14
+[IMPORTANT]
+====
+If you used the Technology Preview OCI feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the oc-mirror OCI feature to download Operator catalogs from a registry and them save to disk in OCI format.
```

Comment:
```suggestion
====
If you used the Technology Preview OCI feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the oc-mirror OCI feature to download Operator catalogs from a registry and then save to disk in OCI format.
```

Reply:
Oof good catch thanks.

However, should I replace this statement with your suggested wording in the other file?

```suggestion
====
If you used the Technology Preview OCI feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the OCI feature of the oc-mirror plugin to create an image set archive as a first step to mirroring to a fully disconnected cluster.
```

Reply:
Yes, good idea. 
I'm hoping we'll have concensus on a better flag/feature name soon, so that it also helps the general understanding.